streamlit_web.py

```diff
@@ -60,8 +62,7 @@ def main():
         if debug_mode:
             st.write(main_df)
         if analysis_mode == "Home":
-            with open("README.md") as readme_file:
-                st.markdown(readme_file.read())
+            landing_pg()
```

Comment:
```suggestion
            readme()
```
This should stay as `readme()` otherwise it might give two select boxes in the sidebar.